hadoop-hdfs-project/hadoop-hdfs-httpfs/pom.xml

```diff
@@ -255,6 +255,7 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <testFailureIgnore>true</testFailureIgnore>
```

Comment:
make this a property set in hadoop -projecvt; defaut true. then we can -D it for this process

Reply:
Okay will update

Reply:
Done.